src/EventStore.Common.Tests/EventStore.Common.Tests.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+
+        <IsPackable>false</IsPackable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0" />
```

Comment:
including a reference to this test logger is handy (or maybe essential) for the CI

```
<PackageReference Include="GitHubActionsTestLogger" Version="2.0.1" />
```

also tabs to indent rather than spacing is most conventional in the codebase